src/module/apps/world-clock/app.ts

```diff
@@ -8,19 +9,56 @@ interface WorldClockData {
     date: string;
     time: string;
     options?: object;
-    user: User;
+    user: typeof game.user;
```

Comment:
```suggestion
    user: UserPF2e;
```
This is a deeper type error we need to fix, but let's not hide it away like that.